### PR TITLE
Music bot play skip stop functions

### PR DESCRIPTION
**Pull request by Mert18**
Music bot starter. Created music folder under commands, other music bot functionalities will be added here.

### CODE REVIEW

**Comment by j-dogcoder** [comment]
Hey @Mert18! Please make sure to have PR's into the **staged** branch as directed by our [contributing.md](https://github.com/Heptagram-Bot/Heptagram/blob/master/CONTRIBUTING.md)

**Comment by j-dogcoder** [comment]
Also, I got the below message when running yarn install. Is that something to be concerned about?
Screen Shot 2021-07-06 at 9 16 30 AM

File: commands/music/play.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { replies } = require('../../config.json');
+
+module.exports = {
+	name: 'play',
+	description: 'Plays the given song.',
+	execute: async ({ client, message, args }) => {
+		if (!message.member.voice.channel) return message.channel.send('🔴 **Please enter a voice channel to play a song.**');
+		if (message.guild.me.voice.channel && message.member.voice.channel.id !== message.guild.me.voice.channel.id) return message.channel.send(replies.sameVC);
+
+		const music = args.join(' ');
```
Comment (thinkverse) [inline commands/music/play.js:10]: What if no URL is provided? Best to add a help message if `args` is empty.
Something along the lines of; `Please provide a YouTube URL to play`.

**Review by thinkverse — CHANGES_REQUESTED** [review]

**Review by j-dogcoder — CHANGES_REQUESTED** [review]
Thank you for the PR @Mert18! I have a few things.

1. Can these messages be changed to embeds? Most of the bot uses them, and they just look nicer.
2.  Currently the bot reacts to any command it successfully executes, with a ✅. However, It would be great if when in the music commands the user is told something like "you must be in a voice channel", that we could take away the checkmark and replace it with the ❌? [This](https://discordjs.guide/popular-topics/reactions.html#removing-reactions-by-emoji) may be a useful resource for doing that.
3. I try to keep the main.js file as clean as possible. Is there any way to move some of the music bot stuff into a new file? I understand if it isn't, I am just trying to keep things tidy.

Thanks!

**Comment by Mert18** [comment]
> Thank you for the PR @Mert18! I have a few things.
> 
> 1. Can these messages be changed to embeds? Most of the bot uses them, and they just look nicer.
> 2. Currently the bot reacts to any command it successfully executes, with a ✅. However, It would be great if when in the music commands the user is told something like "you must be in a voice channel", that we could take away the checkmark and replace it with the ❌? [This](https://discordjs.guide/popular-topics/reactions.html#removing-reactions-by-emoji) may be a useful resource for doing that.
> 3. I try to keep the main.js file as clean as possible. Is there any way to move some of the music bot stuff into a new file? I understand if it isn't, I am just trying to keep things tidy.
> 
> Thanks!

Okay I'll do my best :)

**Comment by j-dogcoder** [comment]
@Mert18 What are the little problems?

**Comment by Mert18** [comment]
First, I have some trouble using Git system since I'm relatively new to Open Source. So forgive me for any problem that I cause :)

All the comments are embeds now.
Replacing emojis with ❌ is a bit problematic. Since ✅ is added after the play.js runs, I can't remove it in play.js file. I'm open to any suggestions.
I tried to move music configurations from main.js file, event handler directories is an option but I wasn't sure.

About Distube npm package, you said you're getting an update error, there is no kind of error in npm, also I checked the official page, there are no signs of it.

**Comment by Mert18** [comment]
Oh also, I use the same lines of code in play.js skip.js and stop.js for error handling. When the user is not in a voice channel, error shows up. It seems repetitive, we must find a way to avoid this.

**Comment by j-dogcoder** [comment]
In terms of being new to git, you'll get used to it quickly! I am hoping that maybe someone from our @Heptagram-Bot/open-source-contributors team could help with the reactions? I have sorted the distube npm error, so that is all fixed. I have no idea about the error. Perhaps if you provide it I can take a closer look? Maybe someone from the team wants to help with that?

**Comment by Mert18** [comment]
The error is showing up in embeds, saying You are not in a voice channel (as we expected). But the problem is, the code is repetitive in those three files.

**Comment by j-dogcoder** [comment]
That's fine!

**Comment by j-dogcoder** [comment]
Is it just emojis left?

**Comment by Mert18** [comment]
Yes. If the command is invalid, the bot react with ❌ but then also reacts with ✅

**Review by j-dogcoder — APPROVED** [review]
This is done for the most part, so I am going to approve it. We can work on emojis and other kinks as needed.

**Pull request merged**